File: java/src/n03_animal_race/Antelope.java
package n03_animal_race;

public class Antelope extends Animal {
    public Antelope() {
        super("Antelope", 20, 80, 40);
    }

    @Override
    public void makeNoise() {
        System.out.println("snort");
    }

    public void jump() {
        System.out.println("yeeeee");
    }
}

File: dart/src/n03_race/antelope.dart
import 'animal.dart';

class Antelope extends Animal {
  Antelope() : super('Antelope', 20, 80, 40);

  @override
  void makeNoise() {
    print('snort');
  }
  
  void jump() {
    print('yeeeee');
  }
}
